src/pages/_app.tsx

```diff
@@ -11,13 +11,21 @@ import '@fontsource/metropolis/700.css';
 import { config } from '@fortawesome/fontawesome-svg-core';
 import AOS from 'aos';
 import 'aos/dist/aos.css';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import Script from 'next/script';
 import Head from 'next/head';
+import CookieModal from '@/components/CookieModal';
 
 config.autoAddCss = false;
 
 export default function App({ Component, pageProps }: AppProps) {
+  const [storedCookie, setStoredCookie] = useState<string | null>(null);
+
+    useEffect(() => {
+        const userCookie = localStorage.getItem('userCookie')
+        setStoredCookie(userCookie);
+    }, []);
```

Comment:
What happen when user Do a reject, will this re-eval at that time?

Reply:
No. It won't. But it should to get the latest value.

Reply:
Updated